content/integrate/redis-mcp/client-conf.md

```diff
@@ -124,6 +125,16 @@ configuration as shown below:
 }
 ```
 
+For Augment in VS Code or JetBrains IDEs, you can also add the configuration
+from the Settings panel. Open the **Settings** panel using the menu
```

Comment:
```suggestion
from the Settings panel. 

Open the **Settings** panel using the menu
```
This is kind of a nitpick, but I'd recommend splitting this paragraph to make it a little easier to scan.